chore(readme): add react-scripts fix to readme (#416)
smooth-code/jest-puppeteer: README.md

}
```

> **NOTE**: Be sure to remove any existing `testEnvironment` option from your Jest configuration. The `jest-puppeteer` preset needs to manage that option itself. 

Reviewer: Why was the blank line removed after this note?

```suggestion
**NOTE**: Be sure to remove any existing `testEnvironment` option from your Jest configuration. The `jest-puppeteer` preset needs to manage that option itself.

```

})
```

> If you are using Typescript, you will need to include the following above imports:

Reviewer: This seems to be an issue with `react-scripts` not typescript.

```suggestion
If you are using `react-scripts`, you will need to include the following above imports:
```

})
```

> If you are using `react-scripts`, you will need to include the following above imports:
> 
> ```js
> /**
>  * @jest-environment puppeteer
>  */
> ```
> 
> or alternatively pass the env via command line:
> 
> ```js
>   "web:test": "react-app-rewired test --env=puppeteer",
> ```
> 
> 

Reviewer: I think the command line should be standard way so people don't need to add the comment on every test file.

```suggestion
If you are using `react-scripts`, you will need to pass the environment via command line:

```js
  "test": "react-scripts test --env=puppeteer",
```

or alternatively include the following comment at the top of each test file:

```js
/**
 * @jest-environment puppeteer
 */
```

```
